frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSection.tsx

```diff
@@ -146,51 +253,76 @@ export const AuthSettingsSection = ({
         onKeyDown={handleEnterKeyDown}
         onChange={(e: FormAuthType) => setFocusedValue(e)}
       >
-        {radioOptions.map(([authType, text]) => (
-          <Fragment key={authType}>
-            <Box onClick={handleOptionClick(authType)}>
-              <Radio value={authType} isDisabled={isDisabled(authType)}>
-                <Flex align="center">
-                  {text}
-                  {authType === FormAuthType.SGID ? (
-                    <>
-                      <Tooltip
-                        label={SGID_TOOLTIP}
-                        placement="top"
-                        textAlign="center"
-                      >
-                        <Icon as={BxsHelpCircle} aria-hidden marginX="0.5rem" />
-                      </Tooltip>
-                      <Link
-                        href={OGP_SGID}
-                        isExternal
-                        // Needed for link to open since there are nested onClicks
-                        onClickCapture={(e) => e.stopPropagation()}
-                      >
-                        Contact us to find out more
-                      </Link>
-                    </>
-                  ) : null}
-                  {authType === FormAuthType.CP ? (
-                    <Tooltip
-                      label={CP_TOOLTIP}
-                      placement="top"
-                      textAlign="center"
+        <VStack spacing={8} align="flex-start">
+          <Box onClick={handleOptionClick(FormAuthType.NIL)}>
+            <Radio
+              value={FormAuthType.NIL}
+              isDisabled={isDisabled(FormAuthType.NIL)}
+            >
+              <Flex align="center">{NONE_AUTH_TEXT}</Flex>
+            </Radio>
+          </Box>
+          <Divider />
+          <Box>
+            <SGIDText />
+
+            {sgIDOptions.map(([authTypeStr, text, ariaLabel]) => {
+              const authType = authTypeStr as FormAuthType
+              return (
+                <>
+                  <Box key={authType} onClick={handleOptionClick(authType)}>
+                    <Radio
+                      value={authType}
+                      isDisabled={isDisabled(authType)}
+                      aria-label={ariaLabel}
+                    >
+                      <Flex align="center">{text}</Flex>
+                    </Radio>
+                  </Box>
+                </>
```

Comment:
2 comments for this chunk: 
1. Can remove the additional `<></>` wrapper, it's not needed.
```suggestion
                <Box key={authType} onClick={handleOptionClick(authType)}>
                  <Radio
                    value={authType}
                    isDisabled={isDisabled(authType)}
                    aria-label={ariaLabel}
                  >
                    <Flex align="center">{text}</Flex>
                  </Radio>
                </Box>
```
2. Can you extract this into a functional component? Something like:
```ts
const optionComponent = (authTypeStr, text, ariaLabel) => {
  const authType = authTypeStr as FormAuthType
  return (
    <Box key={authType} onClick={handleOptionClick(authType)}>
      ...
    </Box>
  )
}
```